digitalfabrik/entitlementcard
# 1430: Show extend card notification only if the card is valid

### Short description
Just noticed a small issue:
After a user extends the card and activates the new one, the extend card notification is still displayed for the old (revoked) card, and this can be confusing.

### Proposed changes
Change the condition to display the notification: if the card is valid (instead of not expired) and extendable

### Side effects
Couldn’t think of any

### Testing
If needed, please refer to the detailed testing instructions from here:
https://github.com/digitalfabrik/entitlementcard/pull/1772
Once the extendable card has been created:
- click ‘Pass verlängern’ in the app notification
- submit the form
- activate the new card
- check that the extend card notification is not displayed anymore

### Resolved issues
Follow up for #1430
